ipyvolume/embed.py

```diff
@@ -140,63 +57,118 @@ def save_js(dirname, makedirs=True):
     src = os.path.join(dir_name_src, "index.js")
     shutil.copy(src, dst)
 
-def embed_html(filename, widgets, makedirs=True, copy_js=True,
-               drop_defaults=False, all=False, title="ipyvolume embed example", external_json=False,
-               indent=2,
-               template=template, template_options={"embed_url":ipywidgets.embed.DEFAULT_EMBED_SCRIPT_URL},
-               widget_view_template=widget_view_template, **kwargs):
-    """ output the the ipywidgets to a standalone html file
-    
-    :param filename: filepath to output to
-    :param widgets: list of ipywidget instances
-    :param makedirs: whether to make directories in the filename path, if they do not already exist
-    :param copy_js: whether to copy a local version of the ipyvolume javascript to the same folder
-    
+
+def save_requirejs(filepath='require.min.js', url="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"):
+    """ download and save the require javascript to a local file
+
+    :type filepath: str
+    :type url: str
+    """
+    content = requests.get(url).content
+    with io.open(filepath, 'w', encoding='utf8') as f:
+        f.write(content.decode("utf8"))
+
```

Comment:
done. I've kept with the `requests` package, because its definitely better the urlretrieve and its allowed me to get pretty good (I think), pythonic version of the wget/curl feature: `ipyvolume.utils.download_to_file` is able to download in chunks, resume from partially downloaded files and, better than wget/curl, has a simple progress bar when running in both the console and notebook. Also `download_to_bytes` and `download_yield_bytes` allow files to be downloaded in memory. Give it a go, see what you think 😀